conda/recipes/pyraft/meta.yaml

```diff
@@ -42,6 +43,7 @@ requirements:
     - libraft-headers {{ version }}
     - nccl>=2.9.9
     - rmm {{ minor_version }}
+    - ucx >={{ ucx_version }}
```

Comment:
I think we should remove the `>=` here

Reply:
Please see comment above

Reply:
That said, we can probably drop this line as the feedstock has [`run_exports`]( https://github.com/conda-forge/ucx-split-feedstock/pull/92/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR40-R41 ), which covers adding and pinning this.